internal/server/providers/oidc.go

```diff
@@ -61,16 +60,15 @@ func WithOIDCClient(ctx context.Context, client OIDCClient) context.Context {
 }
 
 type oidcClientImplementation struct {
-	ProviderID   uid.ID
-	Domain       string
-	ClientID     string
-	ClientSecret string
-	RedirectURL  string
+	ProviderModel models.Provider
```

Comment:
```suggestion
```

Looks like this isn't used, but I think accepting this suggestion will require a `gofmt`.

Reply:
good eye, thanks